### ISSUE TITLE
Allow return focus to app while Popover is open

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#778]
Not implemented in #776 

Assistive tech users should be able to exist the trapped/scoped focus of Popover with F6 key and return to the "normal app" which maintaining the dialog being open.

F6 key should move focus between the application and an open non-model dialog. I'm not sure how the active dialog would be stored--if there's only one open we might attach an event to the body to catch F6 and have a global capture. My understanding is that you only want one dialog open at a time. An outside mouse click is going to close the dialog. I'm unsure about a keyboard event.
